include/sdsl/rrr_vector_15.hpp

```diff
@@ -27,7 +27,7 @@ namespace sdsl {
  *  * m_nr_to_bin: 64 kB = (2^15 entries x 2 bytes)
  *  * m_bin_to_nr: 64 kB = (2^15 entries x 2 bytes)
  */
-template<typename T=void>
+template <typename T=void>
```

Comment:
Can you remove this file from the commit? Since there are no changes in this file necessary, I suggest not fixing formatting issues here.

Reply:
This file was not removed yet from the commit